community-operators/jenkins-operator/0.3.0/jenkins-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,326 @@
+              ],
+              "image":"quay.io/openshift/origin-jenkins:latest",
+              "imagePullPolicy":"Always",
+              "livenessProbe":{
+                "httpGet":{
+                  "path":"/login",
+                  "port":8080,
+                  "scheme":"HTTP"
+                },
+                "initialDelaySeconds":420,
+                "periodSeconds":360,
+                "timeoutSeconds":240
+              },
+              "name":"jenkins-master",
+              "readinessProbe":{
+                "httpGet":{
+                  "path":"/login",
+                  "port":8080,
+                  "scheme":"HTTP"
+                },
+                "initialDelaySeconds":3,
+                "periodSeconds":0,
+                "timeoutSeconds":240
+              },
+              "resources":{
+                "limits":{"cpu":"600m","memory":"4Gi"},
+                "requests":{"cpu":"500m","memory":"3Gi"}
+              }
+            }],
+            "securityContext":{"runAsUser":1000700001}},
+            "restore":{"action":{},"containerName":""},
+            "service":{"port":8080,"type":"ClusterIP"},
+            "slaveService":{"port":50000,"type":"ClusterIP"}
+          }
+        }]
+    capabilities: Basic Install
+    categories: Integration & Delivery
+    certified: "false"
+    containerImage: quay.io/redhat-developer/openshift-jenkins-operator:latest
+    description: >-
+      Kubernetes native operator which fully manages Jenkins on Kubernetes.
+    repository: https://github.com/redhat-developer/jenkins-operator
+    support: Red Hat
+  name: jenkins-operator.v0.3.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Jenkins
+      displayName: Jenkins
+      kind: Jenkins
+      name: jenkins.jenkins.io
+      resources:
+      - kind: Deployment
+        name: ""
+        version: v1
+      - kind: Service
+        name: ""
+        version: v1
+      - kind: ReplicaSet
+        name: ""
+        version: v1
+      - kind: Pod
+        name: ""
+        version: v1
+      - kind: Secret
+        name: ""
+        version: v1
+      - kind: ConfigMap
+        name: ""
+        version: v1
+      version: v1alpha2
+  description: |+
+    ##What's the Jenkins Operator?
+    
+    Jenkins operator is a Kubernetes native operator which fully manages Jenkins on Kubernetes. It was built with immutability and declarative configuration as code in mind.
+
+
+    Out of the box it provides:
+
+
+    - Integration with Kubernetes
+
+    - Pipelines as code
+
+    - Extensibility via groovy scripts or configuration as code plugin
+
+    - Security and hardening
+
+    - Problem statement and goals
+
+    The main reason why we decided to implement the Jenkins Operator is the fact that we faced a lot of problems with standard Jenkins deployment. We want to make Jenkins more robust, suitable for dynamic and multi-tenant environments.
```

Comment:
Update description to explain what the operator does in passive terms in contrast to what "we" did :)

Reply:
I will update this in another PR.